### DISCUSSION TITLE
Welcome to LIFX-Control-Panel Discussions!

### DISCUSSION THREAD

**Discussion by samclane** [samclane/LIFX-Control-Panel#118] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mister-Curious** [comment]
Greetings. I am brand new to the smart lighting scene, but am an ambitious fellow with lots of ideas. I am using an older build of LCP (latest build just crashes on startup, I posted about it in the issues section) but keep getting an error on launch about no lights on the LAN. I checked the router settings through my browser and it is showing both lights as being connected. So far I haven't been able to find anything that explains how users might go about connecting their lights to the LAN through the app or any .cfg. 

As far as my setup goes, I just have two LIFX lights that are working just fine with the native IOS and Desktop apps, but I really want to integrate some of your custom functionality.Could you kindly point me to a guide, or perhaps explain how I might go about connecting my lights to my LAN so that the software can recognize my lights?

Thanks!
